infra/aws/terraform/cdn.packages.k8s.io/s3_bucket.tf

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_s3_bucket" "cdn_packages_k8s_io" {
+  bucket = "${var.prefix}cdn-packages-k8s-io-${data.aws_region.current.name}"
+
+  tags = local.tags
+}
+
+resource "aws_s3_bucket_ownership_controls" "cdn_packages_k8s_io" {
+  bucket = aws_s3_bucket.cdn_packages_k8s_io.id
+
+  rule {
+    object_ownership = "BucketOwnerEnforced"
+  }
```

Comment:
Could we instead disable ACLs for the bucket?

Reply:
`BucketOwnerEnforced` means that ACLs are disabled. Here's how it looks like in the AWS dashboard:

![image](https://github.com/kubernetes/k8s.io/assets/18719127/d5e25441-e479-4c02-ae53-52bf8a77346e)

Reply:
Ah, right. Sorry. Maybe add that detail as a comment.